client/packages/common/src/types/schema.ts

```diff
@@ -6831,16 +6856,20 @@ export type PurchaseOrderLineFilterInput = {
 export type PurchaseOrderLineNode = {
   __typename: 'PurchaseOrderLineNode';
   authorisedNumberOfUnits?: Maybe<Scalars['Float']['output']>;
+  comment?: Maybe<Scalars['String']['output']>;
   expectedDeliveryDate?: Maybe<Scalars['NaiveDate']['output']>;
   id: Scalars['String']['output'];
   item: ItemNode;
   lineNumber: Scalars['Int']['output'];
+  pricePerUnitAfterDiscount: Scalars['Float']['output'];
+  pricePerUnitBeforeDiscount: Scalars['Float']['output'];
```

Comment:
Are these moving in from another PR?

Reply:
Yeah, I just ran `yarn generate`, so if they're new they probably weren't generated on the base branch.